guides/source/association_basics.md

```diff
@@ -2264,7 +2259,7 @@ The `group` method supplies an attribute name to group the result set by, using
 
 ```ruby
 class Parts < ApplicationRecord
-  has_and_belongs_to_many :assemblies, -> { group "factory" }
+  has_and_belongs_to_many :assemblies, -> { group "factory" }, inverse_of :parts
```

Comment:
Is this (`group` with `inverse_of`) right example?

Reply:
the colon is missing